### PR TITLE
CON-119 Enable admin to invite users as admins

### PR DESCRIPTION
**Pull request by mnswaleh**
## Description ##
This pull request sends email notifications to users:
  - When they are being invited to join Converge
  - When their role is changed

## How can This be tested ##
- Clone the repo: `git clone https://github.com/andela/mrm_api.git`
- Setup project according to `Readme.md`
- checkout to branch `story/CON-119-enable-admin-to-invite-users-as-admins`
- on insomnia
to invite a user to Converge, use their valid Andela email to send the mutation:
 ```
mutation {
  inviteToConverge(email: "user.email@andela.com") {
    email
  }
}
```
to change a user role, use their valid Andela email to send the mutation:
 ```
mutation {
  changeUserRole(email: "user.email@andela.com", roleId: 1) {
    user {
      name
      email
      location
      roles {
        role
      }
    }
  }
}
```

## Type of change ##
Please select the relevant option
- [x] Bug fix(a non-breaking change which fixes an issue)
- [x] New feature(a non-breaking change which adds functionality)
- [ ] Breaking change(fix of a feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist: ##
- [x] My code follows the style guidelines of this project
- [x] I have linted my code prior to submission
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes

## JIRA ##
[CON-119](https://andela-apprenticeship.atlassian.net/browse/CON-119)

## Relevant screenshots: ##
Screenshot 2019-07-10 at 18 22 35
Screenshot 2019-07-10 at 18 20 36

### CODE REVIEW

File: templates/change_role.html [hunk 1]
```diff
@@ -40,12 +40,12 @@ <h5 class="greeting" style="box-sizing: border-box;margin-top: 0;margin-bottom:
...
+                    <a href={{domain}}> <button type=" button" class="btn btn-primary" style="box-sizing: border-box;border-radius: 4px;margin: 0;font-family: 'DINPro', 'Roboto', sans-serif;font-size: 1rem;overflow: visible;text-transform: none;-webkit-appearance: button;display: inline-block;font-weight: 400;text-align: center;white-space: nowrap;vertical-align: middle;-webkit-user-select: none;-moz-user-select: none;-ms-user-select: none;user-select: none;border: 1px solid transparent;padding: .375rem .75rem;transition: color .15s ease-in-out,background-color .15s ease-in-out,border-color .15s ease-in-out,box-shadow .15s ease-in-out;color: #fff;background-color: #007bff;border-color: #007bff;height: 44px;width: 174px;">Login
```
Comment (rajeman) [inline templates/change_role.html]: @mnswaleh The `Login to view` button is not clickable. You can look into this.

Comment (mnswaleh) [reply templates/change_role.html]: Did you configure `DOMAIN_NAME` on your `.env` @rajeman

Comment (rajeman) [reply templates/change_role.html]: Sure I did. Try hitting the mutation within docker to confirm.

Comment (mnswaleh) [reply templates/change_role.html]: let me check into that then

Comment (cop1fab) [reply templates/change_role.html]: @mnswaleh This a great job. The query that changes the user's role works perfectly, however, the one that invites them to join converge return a successful response but does not send an email. Please help me understand if I have the wrong set up or if there is any problem.

File: api/user/schema.py [hunk 2]
```diff
@@ -216,6 +215,8 @@ class Arguments:
 
     @Auth.user_roles('Admin')
```
Comment (vic3king) [inline api/user/schema.py]: please add the `Super_Admin` user role here as super admin should be ab;e to perform this task

**Pull request merged**